nav2_route/src/graph_loader.cpp

```diff
@@ -32,7 +32,9 @@ GraphLoader::GraphLoader(
   route_frame_ = frame;
 
   nav2_util::declare_parameter_if_not_declared(
-    node, "graph_filepath", rclcpp::ParameterValue(std::string("")));
+    node, "graph_filepath", rclcpp::ParameterValue(
+      ament_index_cpp::get_package_share_directory("nav2_route") +
```

Comment:
Remove for production? I think this can be empty due to having a service to change?